stracker/customers/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```

Comment:
CustomUserAdmin yazılmalı! CustomUser oluşturduğumuz için admin paneli üzerinden yönetirken hatalara sebebiyet vermemek için, CustomUser yazıldığında diğer modellerin aksine Admin class'ıyla birlikte yazılıp push'lanmalı.

Reply:
Kesinlikle katılıyorum. Ne var ki, user metodu ile ilgili ne yapılacağından emin olmadığım için eklememiştim. Bir sonraki commitimde düzeltilmiş olacak.